BookingProject/BookingWebAPI/AuditWebApi/appsettings.json

```diff
@@ -16,5 +16,8 @@
   "MongoDbSettings": {
     "ConnectionString": "mongodb://localhost:27017/",
     "DatabaseName": "Audit"
+  },
+  "ApiSettings": {
+    "AccountUserInfo": "https://localhost:7255/api/Account/UserInfo/"
```

Comment:
Just define BaseUrl, and then per each Endpoint u will have a Method in your service